src/constraints/load_balance.jl

```diff
@@ -52,7 +53,9 @@ function add_elec_load_balance_constraints(m, p; _n="")
                 + m[Symbol("dvCurtail"*_n)][t, ts] 
             for t in p.techs.elec)
             + sum(m[Symbol("dvGridToStorage"*_n)][b, ts] for b in p.s.storage.types.elec)
+            + sum(m[Symbol("dvThermalProduction"*_n)][t, ts] / p.cop[t] for t in p.techs.cooling)
             + p.s.electric_load.loads_kw[ts]
+            - p.s.cooling_load.loads_kw_thermal[ts] / EXISTING_CHILLER_COP
```

Comment:
Should the subtraction of `cooling_load.loads_kw_thermal` from the electric load balance only happen when using a DoE reference profile? (What if a user provides their own `electric_load.loads_kw` for example, but wants to use DoE `cooling_load`? Or provides both the `electric_load.loads_kw` and `cooling_load`?)